float64 accXmg
float64 accYmg
float64 accZmg
float64 gyroXdeg
float64 gyroYdeg
float64 gyroZdeg

